include/mata/utils/synchronized-iterator.hh

```diff
@@ -162,7 +162,7 @@ public:
     }
 
     /// Returns the vector of current positions.
-    std::vector<Iterator> get_current() {
+    std::vector<Iterator> get_current() const {
```

Comment:
Can you perhaps remove the comments below? They are probably a nonsense.

Reply:
removed